### ISSUE TITLE
Add keyboard shortcuts to test run execution page

### ISSUE THREAD

**Issue by okainov** [kiwitcms/Kiwi#517]
Currently  you need 4-5 clicks to execute each test case run. It may be improved a bit by pre-expanding all test case runs first, but then there is still one click to hide or scrolling required.
![image](https://user-images.githubusercontent.com/918446/45139150-28668280-b1af-11e8-83c9-91c54d02887b.png)

It would be nice to have keyboard shortcuts and possibility to go from each test case run to other using arrows, for example. See JIRA with list of issues, there it's possible to navigate via arrows and execute commands by shortcuts

Suggested behavior:
When click "p"\1\5\whatever hotkey, current test case run status is updated (to "passed" in this case), current test case run is collapsed, focus switches to the next case run, next case run is expanded.

**Comment by spencerwhyte** [comment]
The UX of this has been improved greatly since this issue was opened. I believe that the number of clicks has been reduced from four, to just one for our workflow. Thanks team!

We still have a desire for keyboard shortcuts. @atodorov, is this something that should be implemented as a plugin, or is this core kiwi material?

Either way, I'm guessing that the team is waiting on more user feedback before jumping on this.

**Comment by atodorov** [comment]
> The UX of this has been improved greatly since this issue was opened. I believe that the number of clicks has been reduced from four, to just one for our workflow. Thanks team!
> 
> We still have a desire for keyboard shortcuts. @atodorov, is this something that should be implemented as a plugin, or is this core kiwi material?
> 

I'm not sure Kiwi TCMS has the capability for UI plugins. We've done work only on backend plugins till now. Feel free to give it a try and open issues/pull requests as necessary. 

> Either way, I'm guessing that the team is waiting on more user feedback before jumping on this.

This isn't immediate priority given that it has received minimum user feedback/upvotes, given the UI is usable as-is and given that it's not part of existing roadmap. 

It is still a valid request just not on our radar ATM.

**Comment by atodorov** [comment]
Not going to fix ATM. Extremely low priority. Pull request is welcome.
